bin/target_user.sh

```diff
@@ -8,29 +8,33 @@
 #
 log_src='['${0##*/}']'
 
+odoo_user='odoo'
+
 # Check if there's a target user to run Odoo
 if [ "$TARGET_ID" ]; then
-    echo $log_src[`date +%F.%H:%M:%S`]' TARGET_ID = '$TARGET_ID
-
     echo $log_src[`date +%F.%H:%M:%S`]' Check if target user exists...'
-    EXISTS=$( getent passwd $TARGET_ID | wc -l )
-    echo $log_src[`date +%F.%H:%M:%S`]' EXISTS = '$EXISTS
+    exists=$( getent passwd $TARGET_ID | wc -l )
 
     # Create target user
-    if [ $EXISTS == "0" ]; then
-        echo $log_src[`date +%F.%H:%M:%S`]' Create target user...'
-        adduser --uid $TARGET_ID --disabled-password --gecos "" --shell=/bin/bash target-user
+    if [ $exists == "0" ]; then
+        echo $log_src[`date +%F.%H:%M:%S`]' Create target Odoo user...'
+        odoo_user='target-odoo-user'
+        adduser --uid $TARGET_ID --disabled-password --gecos "" --shell=/bin/bash $odoo_user
 
         # Add target user to odoo group so that he can read/write the content of /opt/odoo
         echo $log_src[`date +%F.%H:%M:%S`]' Add target user to odoo group...'
-        usermod -a -G odoo target-user
-    # If the user already exists, check if it's the same as odoo
+        usermod -a -G odoo $odoo_user
     else
         echo $log_src[`date +%F.%H:%M:%S`]' Target user already exists, make sure it is odoo...'
-        ODOO_ID=$( id -u odoo )
-        if [ $TARGET_ID -ne $ODOO_ID ]; then
+        odoo_user_id=$( id -u $odoo_user )
+
+        # If the user already exists, check if it's the same as odoo
+        if [ $TARGET_ID -ne $odoo_user_id ]; then
             echo $log_src[`date +%F.%H:%M:%S`]' The target user is not the same as odoo, exiting'
```

Comment:
No, switching users is not an option, since it will raise a `chown` on all the files inside `/opt/odoo` which is the home folder of `odoo` user. This is exactly what we want to avoid.
As a reminder, first version of the user mapping was using a `chown` of `odoo`'s home folder and that resulted in extremely annoying latency when starting the container. The solution implemented aims to workaround that issue.
FYI in this PR the target user mapping system has not evolved (there's just a bit of refactoring), it all comes from #56 where this solution has been discussed and approved. There's just a bug to fix in the case mentioned here (mapping to the same user as `odoo`).